Source/dom/models/__tests__/CurvePoint.test.js

```diff
@@ -52,8 +87,9 @@ test('should be able to tell if a point is selected)', () => {
   //still no selection
   expect(shape.points[0].isSelected()).toBe(false)
 
-  //select the first point  
-  .eventHandlerManager()
```

Comment:
@acf API tests were failing after I merged develop and I _think_ there was a problem with this line. I fixed it in a way that I'm hoping matches the intention, could you please double-check if it makes sense? Thanks

Reply:
It does 🙈